cgroup_top.py

```diff
@@ -214,6 +217,8 @@ def type(self):
             return 'systemd'
         elif path == '/user.slice' or path == '/system.slice' or path.startswith('/system.slice/'):
             return 'systemd'
+        elif regexp_ovz_container.match(path) and path != '/0' and HAS_OPENVZ:
```

Comment:
This partially mitigates the above comment if we can safely assume that container system won't be mixed. I'll need to get additional input.